demo/admin/src/products/ProductsGrid.tsx

```diff
@@ -210,6 +211,14 @@ export function ProductsGrid() {
                 );
             },
         },
+        {
+            field: "manufacturer",
+            headerName: intl.formatMessage({ id: "products.manufacturer", defaultMessage: "Manufacturer" }), // shown in filter-column-list and find-column-list
+            sortable: false,
+            valueGetter: ({ value }) => (value ? value.name : "-"),
```

Comment:
valueGetter should not return "-", that would be something for valueFormatter. And I would skip the "-" anyway.

```suggestion
            valueGetter: ({ value }) => (value?.name),
```
or
```suggestion
            valueGetter: ({ row }) => (row.value?.name),
```